source/tutorial/routes-and-templates.md

```diff
@@ -155,9 +155,9 @@ installing route-test
   create tests/unit/routes/index-test.js
 ```
 
-Unlike the other route handlers we've made so far, the `index` route is special: it does NOT require an entry in the router's mapping, and you'll notice that the action of updating the router was not taken by the generator. We'll learn more about why the entry isn't required when we touch on nested routes.
+Unlike the other route handlers we've made so far, the `index` route is special: it does NOT require an entry in the router's mapping, this means that the action of updating the router was not taken by the generator. We'll learn more about why the entry isn't required when we touch on nested routes.
```

Comment:
I'd just remove _this means that the action of updating the router was not taken by the generator_.